node-repository/src/test/java/com/yahoo/vespa/hosted/provision/autoscale/AutoscalingTester.java

```diff
@@ -152,6 +152,37 @@ public void addCpuMeasurements(float value, float otherResourcesLoad,
         }
     }
 
+    /**
+     * Adds measurements with the given resource value and ideal values for the other resources,
+     * scaled to take one node redundancy into account.
+     * (I.e we adjust to measure a bit lower load than "naively" wanted to offset for the autoscaler
+     * wanting to see the ideal load with one node missing.)
+     *
+     * @param otherResourcesLoad the load factor relative to ideal to use for other resources
+     * @param count the number of measurements
+     * @param applicationId the application we're adding measurements for all nodes of
+     */
+    public void addDiskMeasurements(float value, float otherResourcesLoad,
+                                    int count, ApplicationId applicationId) {
+        NodeList nodes = nodeRepository().nodes().list(Node.State.active).owner(applicationId);
+        float oneExtraNodeFactor = (float)(nodes.size() - 1.0) / (nodes.size());
+        for (int i = 0; i < count; i++) {
+            clock().advance(Duration.ofMinutes(5));
+            for (Node node : nodes) {
+                Load load = new Load(ClusterModel.idealQueryCpuLoad * otherResourcesLoad,
+                                     value,
```

Comment:
This adds `value` measurement for memory, not disk?

Reply:
yes, thanks!